metastore.go

```diff
@@ -240,6 +238,24 @@ func (tx *metatx) createSeries(database, name string, tags map[string]string) (*
 	return s, nil
 }
 
+func (tx *metatx) dropSeries(database, name string, seriesID uint32) error {
+	measurmentBucket := tx.Bucket([]byte("Databases")).Bucket([]byte(database)).Bucket([]byte("Series")).Bucket([]byte(name))
+
+	c := measurmentBucket.Cursor()
+
+	for k, _ := c.First(); k != nil; k, _ = c.Next() {
```

Comment:
Also, drop series should take a collection of seriesIDs. That way you can delete many all in a single bolt transaction